generated/proto/index/index.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+package index;
+
+message IndexInfo {
+  message SegmentFileInfo {
+    string fileType = 1;
+  }
+
+  message SegmentInfo {
+    string type = 1;
```

Comment:
nit:s/type/segmentType to be extra clear

Reply:
Sure thing.